proofread docs: for_local.md (#1565)
juicedata/juicefs: docs/en/getting-started/installation.md

- [mingw-w64](https://www.mingw-w64.org/downloads/)

> The latest version is provided by the Linux distribution package manager. Take an example of Ubuntu 20.04+: `mingw-w64` can be installed as follows.

Reviewer: ```suggestion
The latest version could be installed from software repositories on many Linux distributions. Take an example of Ubuntu 20.04+: `mingw-w64` can be installed as follows.
```

software repository 更地道一些，「从包管理器安装」本身是不太地道的说法

Author: ```suggestion
The latest version can be installed from software repositories on many Linux distributions. Take an example of Ubuntu 20.04+: `mingw-w64` can be installed as follows.
```

## Uninstall

> The JuiceFS client only has one binary file, so it can be easily deleted once you find the location of the program. For example, to delete the client that is installed on the Linux system as described above, you only need to execute the following command:

Reviewer: ```suggestion
The JuiceFS client has only one binary file, so it can be easily deleted once you find the location of the program. For example, to uninstall the client that is installed on the Linux system as described above, you only need to execute the following command:
```
